tests/raftstore_cases/test_conf_change.rs

```diff
@@ -613,12 +616,20 @@ fn test_learner_conf_change<T: Simulator>(cluster: &mut Cluster<T>) {
     pd_client.must_add_peer(r1, new_learner_peer(4, 10));
     must_get_equal(&engine_4, b"k2", b"v2");
 
+    // Can't transfer leader to learner.
+    pd_client.transfer_leader(r1, new_learner_peer(4, 10));
+    pd_client.must_leader(r1, new_peer(1, 1));
```

Comment:
The check won't panic even if transfer leader succeeds in the end.

Reply:
Fixed.